src/modules/ekf2/EKF2.cpp

```diff
@@ -225,20 +225,23 @@ int EKF2::print_status()
 	return 0;
 }
 
-template<typename Param>
-void EKF2::update_mag_bias(Param &mag_bias_param, int axis_index)
+template<typename Param1, typename Param2>
+void EKF2::update_mag_bias(Param1 &mag_bias_param, Param2 &mag_bias_var_param, int axis_index)
```

Comment:
FYI this is a relatively large amount of template code bloat for fairly simple functionality.

We should refactor this later to make the logic a simple method and handle the param commit separately.